python/ray/tests/test_output.py

```diff
@@ -6,11 +6,12 @@
 
 
 def test_output():
+    # Use subprocess to execute the __main__ below.
     outputs = subprocess.check_output(
         [sys.executable, __file__, "_ray_instance"],
         stderr=subprocess.STDOUT).decode()
     lines = outputs.split("\n")
-    assert len(lines) == 3
+    assert len(lines) == 3, lines
```

Comment:
Didn't actually fix this test, but should get enough information to debug it from this assert now.